src/ExpandableInput/ExpandableInput.js

```diff
@@ -217,9 +218,18 @@ module.exports = kind(
 	*/
 	inputSpotBlurred: function (inSender, inEvent) {
 		var eventType;
+		var touchSupport, checkNode, mousePointerSupport;
```

Comment:
I might rename these vars to something like  `shouldCloseByTouch`, `eventFromDescendant`, `shouldCloseByPointer`, respectively, for clearer semantics.